src/amo/actions/reviews.js

```diff
@@ -3,12 +3,14 @@ import { SET_REVIEW } from 'amo/constants';
 export const setReview = (review, reviewOverrides = {}) => ({
```

Comment:
The first argument, `review`, is an object from the API. It gets denormalized [like this](https://github.com/kumar303/addons-frontend/blob/817985a0faa8d83fba3100bce7f4f1933f8efc2b/src/amo/actions/reviews.js#L12) so the code you suggested won't work (it would add `review.userId` which would do nothing because `review.user.id` is the only property used).